plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -141,7 +142,9 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
       this.setState({ selectedNodeToDeactivate: node });
     } else if (action === "reactivate") {
       NodeMaintenanceActions.reactivateNode(node, {
-        onSuccess: () => {},
+        onSuccess: () => {
+          MesosSummaryActions.fetchSummary();
```

Comment:
What happens with this newly fetched summary?

Reply:
My understanding was that subscribers (nodes table) would be updated.